### PR TITLE
SPCTR-1872 Fix: Form block - Checkbox throw an issue in console when required toggle is ON.

### PR DESCRIPTION
**Pull request by raojip**
### Description
https://brainstormforce.atlassian.net/browse/SPCTR-1872
Fix:- Checkbox throws an issue in the console when the required toggle is ON.

### Screenshots
Here is the Tested Video:-  [https://d.pr/v/oriPwz]

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/forms/style.scss [hunk 1]
```diff
@@ -208,6 +208,7 @@ select::before {
 	text-align: center;
 	line-height: 1.2;
 	content: " ";
+	// visibility: hidden;
```
Comment (RajkiranBagal) [inline src/blocks/forms/style.scss:211]: Do not keep commented code. If it's not needed then remove it.

File: src/blocks/forms/child-blocks/checkbox/save.js [hunk 2]
```diff
@@ -43,7 +43,6 @@ export default function save( props ) {
 							name={ `${ checkboxName }[]` }
 							value={ optionvalue }
 							required={ checkboxRequired }
-							onInvalid="this.setCustomValidity('Please check this box if you want to proceed.')"
```
Comment (RajkiranBagal) [inline src/blocks/forms/child-blocks/checkbox/save.js:46]: Why was this removed?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/forms/child-blocks/checkbox/save.js (hunk 2 above)
Comment (raojip) [reply src/blocks/forms/child-blocks/checkbox/save.js:46]: This is creating an issue in the checkbox

File: src/blocks/forms/style.scss (hunk 1 above)
Comment (raojip) [reply src/blocks/forms/style.scss]: Removed

File: src/blocks/forms/child-blocks/checkbox/save.js (hunk 2 above)
Comment (AbhijitNage123) [reply src/blocks/forms/child-blocks/checkbox/save.js:46]: You are removing code from save.js. Have u tested the backward case? (Creating forms with a checkbox in v2.0.13 and switching to your branch to verify is it throwing any validation issue by keeping auto-block recovery off)

Comment (AbhijitNage123) [reply src/blocks/forms/child-blocks/checkbox/save.js:46]: @raojip ^

Comment (raojip) [reply src/blocks/forms/child-blocks/checkbox/save.js:46]: Added the deprecated file because of block recovery issue.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**